### ISSUE TITLE
conda 설치 오류 질문합니다.

### ISSUE THREAD

**Issue by kkyo9753** [bab-korea/healthcare-ai-contest#15]
kkyo9753@g-contest-gpu014:/kkyo9753$ bash Miniconda3-latest-Linux-x86_64.sh

Welcome to Miniconda3 py311_23.10.0-1

In order to continue the installation process, please review the license
agreement.
Please, press ENTER to continue
>>> 
Miniconda3-latest-Linux-x86_64.sh: line 143: cannot create temp file for here-document: No space left on device

이 오류 해결 어떻게 하나요? 찾아보니 용량 부족이라고 합니다.

**Comment by bab-korea** [comment]
안녕하세요. 2023 구강이미지 합성데이터 헬스케어 AI 경진대회 운영팀입니다.

GPU 14번 서버에 /home/{본 계정아이디 디렉토리} 으로 작업하신 분이 있어서, 디스크 관련 같은 서버 내에 인원들이 영향을 받아, 해당 서버에서 /home/{본 계정아이디 디렉토리} 에서 작업하신 내용물은 모두 절대 경로: /{본 계정아이디} 로 이동되었습니다.

내용 확인 부탁드립니다.

감사합니다.

**Comment by kkyo9753** [comment]
혹시 kkyo9753 디렉토리 초기화 시켜주실 수 있나요? 아직도 오류나서 다시 진행해보겠습니다.

**Comment by kkyo9753** [comment]
네 설치는 정상적으로 완료되었습니다. 근데 주피터 노트북 설치 이후에도 주피터 노트북 창이 켜지지 않아서 직접 링크타고 들어갔는데 그래도 오류가 발생중입니다. 이거는 어떻게 해결해야할까요? 그리고 pytorch 쿠다 설치를 진행하려고 하는데 제가 로컬에서 했던 방법 그대로 사용해서 설치하면 되는거죠?

**Comment by bab-korea** [comment]
안녕하세요. 2023 구강이미지 합성데이터 헬스케어 AI 경진대회 운영팀입니다.

넵, 내부 GPU 서버에서 변경되거나 추가되는 라이브러리를 서버 내부에 직접적으로 설치하지 않고,  기본셋팅 이외의 추가 수정되는 라이브러리를 적용시키시려면 콘다 안에서만 환경셋팅해주시면, 무방할 것 같습니다.

감사합니다.

**Comment by kkyo9753** [comment]
주피터노트북 실행이 안되는 문제는 어떻게해야할까요?

**Comment by JHyoG** [comment]
현재 conda install tensorflow-gpu까지 실행 완료했습니다.
이때 주피터 노트북을 깔려고 하는데 가상환경인 콘다에서 깔아야 하는것 맞나요?

**Comment by kkyo9753** [comment]
궁금한점이 다른분들은 현재 대회 자료실에 올라온 가이드를 따라했을때 문제가 없으신건가요? jupyter notebook 실행에서 생기는 문제를 가진 사람이 소수인지 여쭤봅니다.

**Comment by tooha289** [comment]
@bab-korea 
안녕하세요
conda 설치 시 아래와 같은 문제가 발생합니다.
![image](https://github.com/bab-korea/healthcare-ai-contest/assets/56501202/adfdbc27-70f8-438f-9625-ace0401832f1)
확인 부탁드려요.
